packages/db/src/search/CollectionSearch.ts

```diff
@@ -0,0 +1,92 @@
+import { Knex } from 'knex';
+import Logger from '@cumulus/logger';
+import { CollectionRecord } from '@cumulus/types/api/collections';
+import { BaseSearch } from './BaseSearch';
+import { DbQueryParameters, QueryEvent } from '../types/search';
+import { PostgresCollectionRecord } from '../types/collection';
+import { translatePostgresCollectionToApiCollection } from '../translate/collections';
+import { BaseRecord } from '../types/base';
+
+const log = new Logger({ sender: '@cumulus/db/CollectionSearch' });
+
+interface ApiCollectionRecord extends BaseRecord, PostgresCollectionRecord {
```

Comment:
ApiCollectionRecord can't extend PostgresCollectionRecord, and they have different fields.  `CollectionRecord` is the api type.
PostgresCollectionRecord is the postgres type for collection, we don't need to declare new type since there are no additional fields in query.